### DISCUSSION TITLE
Use tags with ALE

### DISCUSSION THREAD

**Discussion by patrickReiis** [w0rp/ale#4770] [Q&A - Ask for help with problems] [answered]
Hi, I know nothing about text editors and stuff.

I use nvim.

When I ran the command `:ALEGoToImplementation` in a word in my cursor it works fine!

The problem is that it's too boring to keep typing that out, so I thought by using `Ctrl + ]` it would work, but it doesn't. It says:
```
E433: No tags file                                                                                                                                                  
E426: tag not found: UserStore
```

I use Deno, everything works except for this tag thing. Please help.

**Comment by patrickReiis** [comment]
@neersighted

**Comment by patrickReiis** [comment]
@motato1 

Could you help sir? Thanks.

**Reply by motato1** [reply, marked as answer]
I haven't used ALE in a while but it seems like you don't have overridden the default `Ctrl-]` mapping to use the ALE functoin instead.

`nmap Ctrl-] <Plug>(ale_go_to_implementation)` (it's important to use `nmap` not `nnoremap`)

ALE has options to open the implementation in a split window or a new tab. Check `:help ALEGoToImplementation` for more info.

I haven't tested this thoroughly, but I hope this works for you.

**Reply by patrickReiis** [reply]
I had to use `nmap <C-]> <Plug>(ale_go_to_implementation)` with the "<>" symbols and only the letter `C` as opposed to `Ctrl`, aside from that, THANK YOU VERY MUCH!!!
